chapters/ch48.md

```diff
@@ -1,55 +1,57 @@
 
 > 2. Phoneme recognizer: Some linguists believe that there are basic units of sound called "phonemes." For example, the initial "k" sound in "keep" is the same phoneme as the "c" sound in "cake." This system tries to recognize the phonemes in the audio clip.
 
---> _replace THIS LINE by your translation for the above line_
-
+2. Phần nhận diện các âm vị: Các nhà ngôn ngữ tin rằng trong ngôn ngữ có các đơn vị cơ bản gọi là âm vị. Ví dụ, âm bắt đầu "k" trong từ "keep" thì phát âm giống âm của "c" trong từ "cake". Hệ thống này cố gắng để nhận diện các âm vị trong các audio clip
+ 
 > 3. Final recognizer: Take the sequence of recognized phonemes, and try to string them together into an output transcript.
 
---> _replace THIS LINE by your translation for the above line_
+3. Phần nhận dạng cuối cùng: nhận diện các chuổi âm vị, và cố gắng xâu chuổi chúng với nhau thành một output transcript. 
 
 > In contrast, an end-to-end system might input an audio clip, and try to directly output the transcript:
 
---> _replace THIS LINE by your translation for the above line_
+Ngược lại, một hệ thống hoc bắt đầu - kết thúc đọc đầu vào một audio clip, và cố gắng để cho kêt quả trực tiếp đầu ra là bản dịch:
 
 ![img](../imgs/C48_02.png)
 
 > So far, we have only described machine learning "pipelines" that are completely linear: the output is sequentially passed from one staged to the next. Pipelines can be more complex. For example, here is a simple architecture for an autonomous car:
 
---> _replace THIS LINE by your translation for the above line_
+Từ đầu đến giờ, chúng ta mới mô tả các "chuỗi thuật toán" sử dụng trong học máy hoàn toàn tuyến tính: đầu ra được chuyền đổi theo dạng chuổi từ một trạng thái sang trạng thái tiếp theo. Các "chuỗi thuật toán" này có thể phức tạp hơn. Ví dụ dưới đây là mọt pipline cho một kiến trúc đơn giản của hệ thống lái xe tự động:
 
 ![img](../imgs/C48_03.png)
 
 > It has three components: One detects other cars using the camera images; one detects pedestrians; then a final component plans a path for our own car that avoids the cars and pedestrians.
 
---> _replace THIS LINE by your translation for the above line_
+Kiến trúc này gồm ba phần: phần một phát hiện các xe khác trên đường sử dụng các hình ảnh từ máy quay; phần hai phát hiện người đi bộ; và phần cuối cùng vạch ra đường đi cho chiếc xe để tránh các xe khác cũng như những người đi bộ trên đường. 
 
 > Not every component in a pipeline has to be learned. For example, the literature on "robot motion planning" has numerous algorithms for the final path planning step for the car. Many of these algorithms do not involve learning.
 
---> _replace THIS LINE by your translation for the above line_
+Không phải mọi phần trong một chuỗi thuật toán phải được học. Ví dụ, các nghiên cứu về "quỹ đạo chuyển động robot" sử dụng rất nhiều thuật toán cho bước cuối cùng, bước xác định quỹ đạo đường đi của chiếc xe. Rất nhiều thuật toán này không liên quan đến việc học.   
```

Comment:
"~~quỹ đạo~~ _hoạch địch_ chuyển động robot".
"Rất nhiều _trong số các_ thuật toán này không ~~liên quan đến~~ _bao gồm_ việc học. "